dvc/repo/experiments/refs.py

```diff
@@ -23,9 +23,7 @@ class ExpRefInfo:
 
     namespace = EXPS_NAMESPACE
 
-    def __init__(
-        self, baseline_sha: Optional[str] = None, name: Optional[str] = None
```

Comment:
@pmrowla Q: why do need it to be Optional? What is the case when it's None?

Reply:
@pmrowla @karajan1001 gentle reminder :) please let me know if have an answer to this .... what kind of values do we expect to see for expref?

Reply:
I never met a value of `None` with both of them in the past.

Reply:
@karajan1001 @pared what would be the case for `baseline_sha` to be None?

Reply:
> @karajan1001 @pared what would be the case for `baseline_sha` to be None?

Is it @pmrowla instead of Pawel? I think only if the experiment didn't on any of the commit, but it is not allowed for now.

Reply:
yes, sorry @pared for the noise :(

@pmrowla @karajan1001 I want to disable it for now, if you don't mind? make it always required (it helps with proper types check in the PR). wdyt?

Reply:
I agreed. I don't know in which case these values can be None.

Reply:
We can remove the `Optional` case now. IIRC at one point earlier in the exp development stage there was a case where we had bare exp refs that weren't necessarily tied to a baseline/parent